### DISCUSSION TITLE
Berechtigung erteilen zum durchsuchen der Spotify Biblothek nicht möglich

### DISCUSSION THREAD

**Discussion by zinnik1** [BimmerGestalt/AAIdrive#841] [Q&A]
Hallo ich kann den Button in Aaidrive nicht mehr benutzen um meine Spotify Biblothek im Auto id6 Bmw zu durchsuchen.
Ich habe alle apps aaidrive, spotify und my bmw schon deinstalliert. Aber es geht einfach nicht. Weiss jemand woran es liegen kann. Wenn ich auf den Button erteile klicke passiert einfach nichts. Es kam von einem auf den anderen Tag ohne was zu machen.

**Comment by hufman** [comment]
Unfortunately AAIdrive has little control over Spotify's authentication and library browse functionality. Do you have cell service when connecting to Spotify? Can you browse the Spotify library from the AAIdrive phone UI while at home on wifi? Does AAIdrive think it has permissions to browse the Spotify library (which confirms a successful connection)?

**Reply by wanderschikanentv** [reply]
Same issue here since a few days. It is not possible anymore to use the "Erteilen" button next to "Möglichkeit, deine Spotify-Bibliothek zu durchsuchen". I can see in the Account section of Spotify that AAiDrive does have access to my Spotify account. However, within AAiDrive,  when I click on Spotify in the Musik section, and then go to "Now playing", there is a small information "i" - clicking on it, it says "Spotify App Remote letzter Fehler: UserNotAuthorizedException{"message":"Explicit user authorization is required to use Spotify. The user has to complete the auth-flow to allow the app to use Spotify on their behalf"} and also "Spotify Web API authorisiert!".

**Reply by vsochna-commits** [reply]
Is there any solution to this problem? I have it anyway

If I select Spotify in Music in AAIdrive (I'm playing music) - I see the song, I can switch between songs, but Browse doesn't show ANYTHING - I think that's why it doesn't work in the car either.

Spotify 9.1.20.1452
AAidrive 1.4.3-13-b2d1395

**Reply by vsochna-commits** [reply]
GRANT - no click

**Reply by vsochna-commits** [reply]
![WhatsApp Image 2026-02-11 at 10 47 18](https://github.com/user-attachments/assets/0c0cf273-6fe8-4fcd-bba4-241ad54b9b4a)
![WhatsApp Image 2026-02-11 at 10 50 19](https://github.com/user-attachments/assets/b0d4e7f3-bf6b-4883-9644-d1a06b2ba13d)
![WhatsApp Image 2026-02-11 at 10 50 19 (1)](https://github.com/user-attachments/assets/1e6f3c37-4650-44fa-a16e-9252404ac490)

**Reply by vsochna-commits** [reply]
![WhatsApp Image 2026-02-11 at 10 55 20](https://github.com/user-attachments/assets/8dbbb25a-44d7-4df3-a772-d79d776f1268)

Spotify Autorize AAIdrive

**Reply by vsochna-commits** [reply]
Version AAIDRIVE 1.3.2.85-4891cd2 is OK !!!

**Comment by hufman** [comment]
Please test [the latest build](https://bimmergestalt.s3.amazonaws.com/aaidrive/builds/androidautoidrive-latest-main-mapbox-sentry-release.apk) which has the updated Spotify SDK. You'll need to uninstall the Google App Store version first.

**Reply by vsochna-commits** [reply]
@hufman : latest version 1.4.3-38-934ed8b is OK !!! The ability to browse the Spotify library - was already active immediately after installation (so I didn't have to turn it on).

Thank you very much for the lightning fast response and willingness!!!
